build/Dockerfile

```diff
@@ -16,29 +16,25 @@ ENV PATH=${PATH}:/usr/local/go/bin
 
 # Build WMCB
 WORKDIR /build/
-RUN git clone --single-branch --branch release-4.6 https://github.com/openshift/windows-machine-config-bootstrapper.git
-WORKDIR windows-machine-config-bootstrapper
+COPY windows-machine-config-bootstrapper .
 RUN make build
 
 # Build hybrid-overlay
 WORKDIR /build/
-RUN git clone --single-branch --branch release-4.6 https://github.com/openshift/ovn-kubernetes/
-WORKDIR ovn-kubernetes/go-controller/
+COPY ovn-kubernetes .
+WORKDIR go-controller/
 RUN make windows
 
 # Build Kubernetes node binaries
 WORKDIR /build/
-RUN git clone --branch release-4.6 https://github.com/openshift/kubernetes
-WORKDIR /build/kubernetes
-# TODO: Checking out commit before go 1.15 became a requirement, revert checkout in https://issues.redhat.com/browse/WINC-460
-RUN git checkout f5121a6a6a02ddfafd2bfbf5201b092dc5097ab0
```

Comment:
How are you pinning to this commit with the sub-module?

Reply:
The submodule added shows the commit it is pinned to, one we add a submodule we need to checkout to that commit by cd into the submodule.

Reply:
Please add the commands you executed to add all the sub-modules in the commit message.